.github/workflows/SampleFlow-test.yml

```diff
@@ -32,3 +33,35 @@ jobs:
         cd ../../..
         ./doc/indent
         git --no-pager diff --exit-code
+
+  build-macos:
+
+    runs-on: macos-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: install dependencies
+      run: |
+        brew install cmake boost doxygen
+        git clone https://github.com/tjhei/numdiff.git
```

Comment:
That's the repository of my friend Timo Heister, but it's not the master repository for the numdiff project. Is that what you had in mind?

Reply:
Oh, that's weird - it's the first thing that comes up when I search for numdiff. Can you point me to the master for the numdiff project?

Reply:
The project lives here: https://www.nongnu.org/numdiff/

You should be able to get it via `wget` from http://download-mirror.savannah.gnu.org/releases/numdiff/ as a .tar.gz file that you can unpack via `tar xvzf file.tar.gz`.